Sources/ETCD/ETCDValue.swift

```diff
@@ -0,0 +1,20 @@
+/// ETCDValue conformance to several standard Swift types.
+import Foundation
+import GRPC
+import NIO
+import NIOCore
+import SwiftProtobuf
+
+public protocol ETCDValue {}
```

Comment:
We need to get something from this actually. In the end we have to convert it to a `Data` so we need to add a `var asData: Data { get }` to the protocol which we can then call in the `get` and `set` methods.

Reply:
Hmm, Is my current ETCDValue correct for the asData for each extension type, e.g. should each type have a public asData function?